### ISSUE TITLE
Add instance variable Path to object Drawing

### ISSUE THREAD

**Issue by mguldner** [mguldner/Architecture_Logicielle_SVG#4]
I'm wondering if we should add a `private Path path` to the object `Drawing`.
Indeed, every Drawing we create has a Path : 
- An empty `Drawing` has a `Path` null
- `Action` (extends `Drawing`) create a new `Drawing` (and so has a `Path`)
- `Operator` (extends `Drawing`) create a new `Drawing` (and so has a `Path`)

@legaultpierre : You were speaking about problem with further extension. I don't see where the problem is... 
In order to insert a `Drawing`, we should use a `Path` to define it. 
Indeed, we need a closed `Path` which defines the `Drawing`, and to check it, a good possibility, I think, is to create an instance variable `Path` in the `Drawing` object.

**Comment by legaultpierre** [comment]
@mguldner 
I took a look to the [Asymptote language](http://asymptote.sourceforge.net/), that is the base of our specifications, and it seems that the insert function is something like this:
<code>clip(Drawing, Path)</code> (your can find it used going `Documentation > Clip > first Latex Example`)
So insert would means that we insert a drawing within a path (ie give the drawing some relative position and size?).
I guess the description given on [H. Grall website](http://grall.name/teaching/softwareArchitecture/2016/project_vectorGraphics.html) is a little bit confusing.  

So, in my opinion there is no need to add a `Path` attribute to drawing, and moreover I would find hard to understand what it would mean in "reality": you cannot find a path in every drawing possible.

I'm ready to discuss more on this subject with a coffee tomorrow or on Friday if you want. =)

**Comment by mguldner** [comment]
After looking at some examples of clipping in Java and SVG, I think that this `clip` function has no link with what we want to do. Indeed, it just enables us to cut some parts of the draw, without enabling us to use it later. This can be seen [here](http://tutorials.jenkov.com/svg/clip-path.html).

In fact : 
- In Java, I managed to clip an area of a drawing, but then I can't paste it (we can get the clip with `getClip` but it's a `Shape` and we just can draw the outline of a `Shape` so that's not what we want.
- In SVG, when we use the `clip` attribute, we just can cut some parts of a drawing but we can't store it to use it later (for what I saw).

I was also looking for an other method to copy and paste. I've done some changes to use `clip` in Java and I can push it if you want (even if the result is not what I think we should have), but in SVG, the `clip` does not seem to the good solution.
